reference/masterfiles-policy-framework/standalone_self_upgrade.markdown

```diff
@@ -0,0 +1,17 @@
+---
+layout: default
+title: standalone_self_upgrade.cf
+published: true
+sorting: 150
+tags: [reference, policy entry, MPF]
+---
+
+`$(sys.inputdir)/standalone_self_upgrade.cf` is an independent policy set entry
+like `promises.cf` and `update.cf`. The policy is executed by an independent
+agent executed from the update.cf policy when the class `trigger_upgrade` is
+defined and the host is not seen to be running the desired version of the agent.
+
+trying to converge to a different version of CFEngine. It is designed for use
```

Comment:
This bit here doesn't make sense to me. "trying to converge..." seems like a sentence fragment/typo?